go-controller/pkg/ovn/namespace.go

```diff
@@ -151,6 +151,9 @@ func (oc *Controller) AddNamespace(ns *kapi.Namespace) {
 		}
 	}
 
+	// Set annotations
+	nsInfo.annotations = ns.GetAnnotations()
```

Comment:
@rcarrillocruz you also need to update the annotations when they change.

Reply:
I added that to updateNamespace, let me know if that looks sane